main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -101,6 +101,8 @@ object DocAst {
 
     case class TryCatch(d: Expr, rules: List[(Symbol.VarSym, Class[_], Expr)]) extends Atom
 
+    case class Throw(d: Expr) extends Atom
```

Comment:
`throw exp` is just a keyword, so instead of a full case class, just make a `def` like with `Discard` or `Untag`